queries.sql

```diff
@@ -1,3 +1,12 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+SELECT * FROM animals WHERE name LIKE '%mon';
+SELECT * FROM animals WHERE date_of_birth > '2016-12-31' AND date_of_birth < '2019-01-01';
```

Comment:
- [ ] Kindly fix this **SELECT * FROM animals WHERE date_of_birth >= '2016-12-31' AND date_of_birth <= '2019-01-01';** 
You have not added the equal sign
or you can make way also 
**SELECT name FROM animals WHERE EXTRACT(year FROM date_of_birth) BETWEEN 2016 AND 2019;**

Reply:
Fixed.